crawl-ref/source/spl-util.cc

```diff
@@ -1344,8 +1344,14 @@ int spell_highlight_by_utility(spell_type spell, int default_colour,
     {
         return COL_FORBIDDEN;
     }
+    // Grey out spells for which you lack experience or spell levels.
+    if (spell_difficulty(spell) > you.experience_level
+             || player_spell_levels() < spell_levels_required(spell))
```

Comment:
The indentation on this hanging `||` doesn't match the one in the previous stanza. I think the previous one reflects "crawl style", such as it is, but if you find evidence that your style is more common in the crawl codebase then you should fix the one above it to match this one.